gnome-extra/nm-applet/nm-applet-1.30.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+GNOME_ORG_MODULE="network-manager-applet"
+
+inherit gnome.org meson xdg
+
+DESCRIPTION="NetworkManager connection editor and applet"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="appindicator modemmanager selinux teamd"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+RDEPEND="
+	>=dev-libs/glib-2.40:2
+	>=app-crypt/libsecret-0.18
+	>=net-libs/libnma-1.10.4
+	>=x11-libs/gtk+-3.10:3
+	>=dev-libs/libgudev-147:=
+	>=net-misc/networkmanager-1.16:=[modemmanager?,teamd?]
+	appindicator? (
+		dev-libs/libappindicator:3
+		>=dev-libs/libdbusmenu-16.04.0
+	)
+	modemmanager? ( net-misc/modemmanager )
+	selinux? ( sys-libs/libselinux )
+	teamd? ( >=dev-libs/jansson-2.7:= )
+
+	virtual/freedesktop-icon-theme
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-libs/libxml2
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+"
+
+src_configure() {
+	local emesonargs=(
+		-Dappindicator=$(usex appindicator ubuntu no)
+		$(meson_use modemmanager wwan)
+		$(meson_use selinux)
+		$(meson_use teamd team)
+		-Dmore_asserts=0
+		-Dld_gc=false
+	)
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+	rm "${D}/usr/share/glib-2.0/schemas/org.gnome.nm-applet.gschema.xml" || die
```

Comment:
The `pkgdiff -b` between libnma 1.10.2 and 1.10.4 actually shows

````diff
+  # This one belongs to nm-applet.
+  # Kept here for now to avoid breaking on upgrades.
+  'org.gnome.nm-applet.gschema.xml'
+]
+  configure_file(
+    input: schema + '.in',
+    output: schema,
+    install_dir: join_paths(nma_datadir, 'glib-2.0', 'schemas'),
+    configuration: schema_conf,
+  )
````
so it might make sense to instead remove the file from libnma?

Reply:
Ok, will be done in next commit